doc/source/serve/multi-app.md

```diff
@@ -49,24 +55,20 @@ grpc_options:
   grpc_servicer_functions: []
 
 applications:
-
 - name: app1
```

Comment:
Should we also change the app name to "classify" and "translate"?

Reply:
Since we include `serve build` as part of this doc and it autogenerates `app1` and `app2`, I think it's OK to leave as `app1` and `app2`